test/elixir_internal_certificate/accounts_test.exs

```diff
@@ -0,0 +1,172 @@
+defmodule ElixirInternalCertificate.AccountsTest do
+  use ElixirInternalCertificate.DataCase
+
+  alias ElixirInternalCertificate.Accounts
+  alias ElixirInternalCertificate.Accounts.{User, UserToken}
+
+  setup do
+    %{user: insert(:user)}
+  end
+
+  describe "get_user_by_email/1" do
+    test "does not return the user if the email does not exist" do
```

Comment:
Let use our new convention `given the email does not exist, does not return the user`

![Screen Shot 2565-06-09 at 5 13 27 PM](https://user-images.githubusercontent.com/6076762/172823715-b55bda7f-fb89-4ccd-bf43-446e36a7d455.png)